google/cloud/pubsub/message.h

```diff
@@ -128,6 +132,8 @@ class Message {
   friend void pubsub_internal::SetAttribute(std::string const& key,
                                             std::string value,
                                             pubsub::Message&);
+  friend std::string pubsub_internal::GetAttributeValue(std::string const& key,
+                                                        pubsub::Message&);
```

Comment:
Q. Why do we prefer free friends rather than member functions for these?

Reply:
`Message` is a public API, and we don't necessarily want to grow the API.  The bias is towards friends in `pubsub_internal`.

Reply:
I see.  Thanks.